modules/sandboxed-containers-uninstalling-kata-runtime.adoc

```diff
@@ -0,0 +1,18 @@
+//Module included in the following assemblies:
+//
+// *disabling-sandboxed-container-workloads.adoc
+
+[id="sandboxed-containers-uninstalling-kata-runtime_{context}"]
+
+= Uninstalling Kata Runtime
+
+This section describes how to remove and uninstall the `kata` runtime and all
+its related resources (for example, CRI-O config, RuntimeClass, and so on) from
+your cluster.
+
+.Procedure
+
+1. Delete the `KataConfig` custom resource using `oc delete kataconfig <KataConfig_CR_Name>`.
```

Comment:
A lot of `oc` steps are in the format:
> 1. Do the thing:
`oc do the thing`

So you might consider that format here.